client/my-sites/stats/site.jsx

```diff
@@ -300,28 +286,21 @@ class StatsSite extends Component {
 							/>
 						</div>
 					</div>
-					{ ! isVip && isAdmin && ! hasWordAds && (
-						<Banner
-							className="stats__upsell-nudge"
-							icon="star"
-							title={ translate( 'Start earning money now' ) }
-							description={ translate(
-								'Accept payments for just about anything and turn your website into a reliable source of income with payments and ads.'
-							) }
-							callToAction={ planSupportsWordAdsInstantFeature ? translate( 'Learn more' ) : null }
-							href={ `/earn/${ slug }` }
-							dismissPreferenceName={
-								planSupportsWordAdsInstantFeature ? `stats-earn-nudge-wordads-${ siteId }` : null
-							}
-							event="stats_earn_nudge"
-							tracksImpressionName="calypso_upgrade_nudge_impression"
-							tracksClickName="calypso_upgrade_nudge_cta_click"
-							showIcon={ true }
-							jetpack={ false }
-							horizontal
-						/>
-					) }
 				</div>
+				<Banner
+					callToAction={ translate( 'Learn more' ) }
+					description={ translate(
+						"Parse.ly Analytics makes it easy to understand the full impact of your content. Measure what's driving awareness, engagement, and conversions."
+					) }
+					dismissPreferenceName="calypso_stats_parsely_banner_dismiss"
+					event="calypso_stats_parsely_banner_view"
+					href="http://parse.ly"
+					iconPath={ parselyIcon }
+					title={ translate( 'Discover more stats with Parse.ly Analytics' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Discover more stats with Cloudflare Analytics' )` *ES Score: 6*